components/scream/src/share/field/field_repository.hpp

```diff
@@ -133,6 +136,16 @@ register_field (const identifier_type& id, const std::string& group_name) {
   register_field<RequestedValueType>(id,group_name_set);
 }
 
+template<typename ScalarType, typename Device>
+template<typename RequestedValueType>
+void FieldRepository<ScalarType,Device>::
+register_field (const identifier_type& id, const std::initializer_list<std::string>& groups_names) {
+  for (auto name : groups_names)
```

Comment:
Since `register_field<T>(id,name)` in turn calls the overload with `std::set`, you're better off creating the set with all names from the initializer list:
```
   register_field<RequestedValueType>(id,std::set<std::string>(groups_name));
```

Reply:
Done.